i18n/fr/docusaurus-plugin-content-docs/version-22.04/resources/known-issues.md

```diff
@@ -35,6 +35,7 @@ ALTER TABLE virtual_metrics MODIFY index_id bigint unsigned;
 
 L'autologin n'est actuellement pas géré pour les pages suivantes :
 
+* **Supervision > Map**
```

Comment:
This not true for MAP legacy. 
Maybe something like 
* **Supervision > Map (legacy excluded)**